go.mod

```diff
@@ -5,9 +5,10 @@ go 1.14
 // Pinned to kubernetes-1.18.3
 require (
 	cloud.google.com/go v0.54.0 // indirect
-	github.com/ViaQ/logerr v1.0.4
+	github.com/ViaQ/logerr v1.0.5
 	github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
 	github.com/emicklei/go-restful v2.12.0+incompatible // indirect
+	github.com/go-logr/zapr v0.2.0 // indirect
```

Comment:
I don't think zap is required anymore. Maybe `go mod tidy`?

Reply:
did that:
```
$ grep -r zapr .
./go.sum:github.com/go-logr/zapr v0.1.0/go.mod h1:tabnROwaDl0UNxkVeFRbY8bwB37GwRv0P8lg6aAiEnk=
./go.sum:github.com/go-logr/zapr v0.1.1 h1:qXBXPDdNncunGs7XeEpsJt8wCjYBygluzfdLO0G5baE=
./go.sum:github.com/go-logr/zapr v0.1.1/go.mod h1:tabnROwaDl0UNxkVeFRbY8bwB37GwRv0P8lg6aAiEnk=
./go.sum:github.com/go-logr/zapr v0.2.0 h1:v6Ji8yBW77pva6NkJKQdHLAJKrIJKRHz0RXwPqCHSR4=
./go.sum:github.com/go-logr/zapr v0.2.0/go.mod h1:qhKdvif7YF5GI9NWEpyxTSSBdGmzkNguibrdCNVPunU=
./vendor/github.com/go-logr/logr/README.md:- **go.uber.org/zap**: [zapr](https://github.com/go-logr/zapr)
./vendor/modules.txt:# github.com/go-logr/zapr v0.2.0
Binary file ./normalization.test matches
./.bingo/opm.sum:github.com/go-logr/zapr v0.1.0/go.mod h1:tabnROwaDl0UNxkVeFRbY8bwB37GwRv0P8lg6aAiEnk=
./.bingo/operator-sdk.sum:github.com/go-logr/zapr v0.1.0/go.mod h1:tabnROwaDl0UNxkVeFRbY8bwB37GwRv0P8lg6aAiEnk=
./.bingo/operator-sdk.sum:github.com/go-logr/zapr v0.1.1 h1:qXBXPDdNncunGs7XeEpsJt8wCjYBygluzfdLO0G5baE=
./.bingo/operator-sdk.sum:github.com/go-logr/zapr v0.1.1/go.mod h1:tabnROwaDl0UNxkVeFRbY8bwB37GwRv0P8lg6aAiEnk=
Binary file ./bin/operator-sdk-v0.18.2 matches
./go.mod:	github.com/go-logr/zapr v0.2.0 // indirect
```